analyzer/debian-license-feeder/src/main/java/eu/fasten/analyzer/debianlicensefeeder/DebianLicenseFeederPlugin.java

```diff
@@ -82,7 +84,7 @@ public void consume(String record) {
 
             } catch (Exception e) { // Fasten error-handling guidelines
                 logger.error(e.getMessage(), e.getCause());
-                setPluginError(e);
+                throw e;
```

Comment:
I see that the exception is not consistently re-thrown across all plugins. What is the effect of that, doesn't this change the control flow in the application?

Reply:
At least for PomAnalyzer, I have just removed that retry mechanism to keep the handling consistent and simple. Throwing the unrecoverable error will also retry to connect to the DB.

Reply:
If I understand it correctly, the error handling (logging and setting error) is now entirely performed in the `FastenKafkaPlugin`. Why don't we just completely get rid of all these `try/catch` blocks in all plugins? These methods only re-throw `e` and do not seem to add any value to the error handling...

Reply:
Seems to be resolved in the last commits